layers/+tools/vagrant/packages.el

```diff
@@ -9,7 +9,7 @@
 ;;
 ;;; License: GPLv3
 (setq vagrant-packages '(vagrant
-                           vagrant-tramp))
+                         vagrant-tramp))
```

Comment:
This change is unnecessary.

Reply:
I thought it was just an obvious indentation fix. So is the original style preferred? It doesn't look very nice to me.